web/pnpm-lock.yaml

```diff
@@ -1184,7 +1207,7 @@ packages:
     peerDependencies:
       eslint: ^6.2.0 || ^7.0.0 || ^8.0.0
       eslint-plugin-vue: ^9.0.0
-      typescript: '#'
+      typescript: '*'
```

Comment:
Why change the `typescript` in this pull request?

Reply:
I didn't do anything else, this just updates automatically when the package is installed